app/models/manageiq/providers/vmware/infra_manager/inventory/parser.rb

```diff
@@ -280,10 +280,7 @@ def parse_pbm_profile(object, _kind, props)
   alias parse_pbm_capability_profile parse_pbm_profile
 
   def parse_pbm_placement_hub(persister_storage_profile, _object, _kind, props)
-    datastore = cache["Datastore"][props[:hubId]]
-    return if datastore.nil?
-
-    persister_storage = persister.storages.lazy_find(parse_datastore_location(datastore))
+    persister_storage = persister.storages.lazy_find(props[:hubId])
```

Comment:
The vc simulators we do our vcr based tests from don't have the SPBM endpoint so this wasn't covered in the spec tests